tianshou/policy/multiagent/mapolicy.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+from typing import Union, Optional, Dict, List
+
+from tianshou.policy import BasePolicy
+from tianshou.data import Batch, ReplayBuffer
+
+
+class RandomMultiAgentPolicy(BasePolicy):
+    """A random agent used in multi-agent learning. It randomly chooses an
+    action from the legal action.
+    """
+
+    def forward(self,
+                batch: Batch,
+                state: Optional[Union[dict, Batch, np.ndarray]] = None,
+                **kwargs) -> Batch:
+        mask = batch.obs.mask
+        logits = np.random.rand(*mask.shape)
+        logits[np.isclose(mask, 0)] = 0
```

Comment:
set to ``0`` is not safe. set to ``-np.inf`` instead. Mask is a strong constraint. However, if there are zeros in logits, this function may return illegal actions.